CONTRIBUTING.md

```diff
@@ -420,6 +420,16 @@ executeSQL("SELECT * FROM {0} LIMIT 2".format(titanic.__genSQL__()), method="fet
   135,
   'Montreal, PQ / Chesterville, ON']]
 ```
+The last thing to know before writing an entire function is the save_to_query_profile() method. It is used to save the method call in a specific Vertica table. This table can be used to run statistics or in order to debug some code.
+
+The function is quite simple and it uses 4 main parameters:
+ - name: it is the name of the function.
+ - path: it is the location of the function. For example, the method corr of the vDataFrame is located in vdataframe.vDataFrame.
+ - json_dict: it is the dictionary of the different parameters.
```

Comment:
It is a guide for the potential future developers, so it is important to be clear. Are all the parameters supposed to appear hear? Or is it up the developer?

Reply:
Corrected